napari/utils/colormaps/bop_colors.py

```diff
@@ -0,0 +1,784 @@
+    [0.4823529411764706, 0.10196078431372549, 0.4823529411764706],
+    [0.48627450980392156, 0.10196078431372549, 0.48627450980392156],
+    [0.48627450980392156, 0.10588235294117647, 0.48627450980392156],
+    [0.49019607843137253, 0.10588235294117647, 0.49019607843137253],
+    [0.49411764705882355, 0.10588235294117647, 0.49411764705882355],
+    [0.49411764705882355, 0.10588235294117647, 0.49411764705882355],
+    [0.4980392156862745, 0.10588235294117647, 0.4980392156862745],
+    [0.4980392156862745, 0.10588235294117647, 0.4980392156862745],
+    [0.5019607843137255, 0.10588235294117647, 0.5019607843137255],
+    [0.5019607843137255, 0.10588235294117647, 0.5019607843137255],
+    [0.5058823529411764, 0.10980392156862745, 0.5058823529411764],
+    [0.5098039215686274, 0.10980392156862745, 0.5098039215686274],
+    [0.5098039215686274, 0.10980392156862745, 0.5098039215686274],
+    [0.5137254901960784, 0.10980392156862745, 0.5137254901960784],
+    [0.5137254901960784, 0.10980392156862745, 0.5137254901960784],
+    [0.5176470588235295, 0.10980392156862745, 0.5176470588235295],
+    [0.5176470588235295, 0.10980392156862745, 0.5176470588235295],
+    [0.5215686274509804, 0.10980392156862745, 0.5215686274509804],
+    [0.5254901960784314, 0.11372549019607843, 0.5254901960784314],
+    [0.5254901960784314, 0.11372549019607843, 0.5254901960784314],
+    [0.5294117647058824, 0.11372549019607843, 0.5294117647058824],
+    [0.5294117647058824, 0.11372549019607843, 0.5294117647058824],
+    [0.5333333333333333, 0.11372549019607843, 0.5333333333333333],
+    [0.5372549019607843, 0.11372549019607843, 0.5372549019607843],
+    [0.5372549019607843, 0.11372549019607843, 0.5372549019607843],
+    [0.5411764705882353, 0.11372549019607843, 0.5411764705882353],
+    [0.5411764705882353, 0.11764705882352941, 0.5411764705882353],
+    [0.5450980392156862, 0.11764705882352941, 0.5450980392156862],
+    [0.5450980392156862, 0.11764705882352941, 0.5450980392156862],
+    [0.5490196078431373, 0.11764705882352941, 0.5490196078431373],
+    [0.5529411764705883, 0.11764705882352941, 0.5529411764705883],
+    [0.5529411764705883, 0.11764705882352941, 0.5529411764705883],
+    [0.5568627450980392, 0.11764705882352941, 0.5568627450980392],
+    [0.5568627450980392, 0.11764705882352941, 0.5568627450980392],
+    [0.5607843137254902, 0.12156862745098039, 0.5607843137254902],
+    [0.5607843137254902, 0.12156862745098039, 0.5607843137254902],
+    [0.5647058823529412, 0.12156862745098039, 0.5647058823529412],
+    [0.5686274509803921, 0.12156862745098039, 0.5686274509803921],
+    [0.5686274509803921, 0.12156862745098039, 0.5686274509803921],
+    [0.5725490196078431, 0.12156862745098039, 0.5725490196078431],
+    [0.5725490196078431, 0.12156862745098039, 0.5725490196078431],
+    [0.5803921568627451, 0.12549019607843137, 0.5803921568627451],
+]
+
+bopd = {"BOPBlue": bop_blue, "BOPOrange": bop_orange, "BOPPurple": bop_purple}
```

Comment:
These will be rendered by the UI, so they should be a bit nicer to read. Additionally, we typically use all lowercase. I suggest:

```suggestion
bopd = {"bop blue": bop_blue, "bop orange": bop_orange, "bop purple": bop_purple}
```